QA/manual_testing.md

```
@@ -368,53 +496,102 @@
+   ```
+
+2. 確認事項:
+   - すべてのコマンド、サブコマンドのヘルプメッセージが適切に表示されること
+   - コマンドの説明が正確であること
+   - オプションの説明が正確であること
+   - 使用例が含まれていること
+
+#### 8.2 コマンド補完のテスト
+
+1. 部分的なコマンド名を入力してタブ補完を試みる
+   
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 3

(MD009, no-trailing-spaces)